Junk4Dollars-appTests/ShowAuctions/AuctionListViewController.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+import UIKit
+import Junk4Dollars_app
+
+class AuctionListViewControllerTests: XCTestCase {
+    func buildController() -> AuctionListViewController {
+        var controller: AuctionListViewController!
+        controller = AuctionListViewController()
+        let tableView = UITableView()
+        controller.auctionTableView = tableView
+        controller.viewDidLoad()
+        return controller
+    }
+
+    func testControllerSetsDataSource() {
```

Comment:
`testControllerSetsTableViewDataSource` would be more accurate.
A lot of things can have data source.